index.js

```diff
@@ -204,6 +204,12 @@ LiveServer.start = function(options) {
 		var proxyOpts = url.parse(proxyRule[1]);
 		proxyOpts.via = true;
 		proxyOpts.preserveHost = true;
+        if (options.unsecureProxy) {
+			process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+            proxyOpts.checkServerIdentity = function () {
```

Comment:
See here the specs for `checkServerIdentity` https://nodejs.org/api/tls.html#tls_tls_connect_options_callback. It's not directly used by `https.request`, but it's used by `tls.connect` which at the bottom end it's being used by `https` (see here: https://github.com/nodejs/node/blob/master/lib/https.js#L83)

I'll add `proxyOpts.rejectUnauthorized  = false;` (that was the main goal of `process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";` but I also think it's better to add both config options)